packages/containers/examples/ExampleList.js

```diff
@@ -44,6 +44,16 @@ const props = {
 	toolbar,
 };
 
+const customHeight = {
+	tile: undefined,
+	large: 200,
+	table: 200,
```

Comment:
Change the value between large and table to demonstrate that the height is per type